### PR TITLE
Profile for spock at ORNL

### PR DESCRIPTION
**Pull request by steindev**
To run on the caar partition.

### CODE REVIEW

**Comment by steindev** [comment]
Not ready for merge yet. Just a few more minutes...

**Review by psychocoderHPC — CHANGES_REQUESTED** [review]

File: etc/picongpu/spock-ornl/caar.tpl [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# host memory per device
+.TBG_memPerDevice=64000
+
+# number of CPU cores to block per GPU
+# we have 16 CPU cores per GPU (64cores/4gpus ~ 16cores)
+.TBG_coresPerGPU=16
+
+# Assign one OpenMP thread per available core per GPU (=task)
+# export OMP_NUM_THREADS=1
```
[the remark this answers is not in the document]
Comment (steindev) [reply etc/picongpu/spock-ornl/caar.tpl]: Oops, correct. :-)

**Review by psychocoderHPC — APPROVED** [review]

**Comment by ax3l** [comment]
@steindev don't forget to link the profile in the docs: https://picongpu.readthedocs.io/en/latest/install/profile.html

**Comment by ax3l** [comment]
Ours is currently at https://warpx.readthedocs.io/en/latest/install/hpc/spock.html, the `HIPCC_COMPILE_FLAGS_APPEND` env var is a good touch.

**Pull request merged**